docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -1185,3 +1185,69 @@ The provided replicas <1>, labels <2> and environment variables <3>  were retain
 If the resource name does not match the application name (or the overridden name) instead of reusing the resource a new one will be added. Same goes for the container.
 If the name of the container does not match the application name (or the overridden name), container specific configuration will be ignored.
 ====
+
+== Service Binding
+
+Quarkus supports binding services to applications via the link:https://github.com/k8s-service-bindings/spec[Service Binding Specification for Kubernetes].
+Specifically Quarkus implements the link:https://github.com/k8s-service-bindings/spec#application-projection[Application Projection] part of the specification, thus allowing
+applications running in appropriately configured Kubernetes clusters to consume services (such as a Database or a Broker) without the need for user configuration.
+
+Currently, the following Quarkus extensions support this feature:
+
+* quarkus-jdbc-mariadb
+* quarkus-jdbc-mssql
+* quarkus-jdbc-mysql
+* quarkus-jdbc-postgresql
+
+This list of extensions will grow as more services with supported bindings become available on Kubernetes.
+
+To enable Service Binding support, in addition to one of currently supported extensions, the `quarkus-kubernetes-service-binding` extension needs to be added to the application dependencies as well as
+having the `quarkus.kubernetes-service-binding.enabled` configuration property set to `true`.
```

Comment:
While this isn't about documentation, I would not have suspected this without docs.

Is there a reason why this isn't on by default when the extension is used?

Reply:
To be honest, I didn't remember this either and on second look it did surprise me too.
Should I change the default to true?

Reply:
Yeah, after all other config sources have higher priority, so… yeah.

Reply:
Fixed